tests/schema/__app/generated/schema.graphql

```diff
@@ -9,7 +9,7 @@ type BatchPayload {
 type Bubble {
   createdAt: DateTime!
   id: String!
-  members(after: UserWhereUniqueInput, orderBy: BubbleMembersOrderByInput, where: BubbleMembersWhereInput): [User!]!
+  members(after: UserWhereUniqueInput, orderBy: [BubbleMembersOrderByInput!], where: BubbleMembersWhereInput): [User!]!
```

Comment:
BREAKING CHANGE to document

orderBy now receives an array of filters

Reply:
I'm confused why we didn't have this change before since list-ordering by multiple fields comes from https://github.com/prisma/prisma/releases/tag/2.4.1 which we already support.